### PR TITLE
Convert ocm account quota to ocm list quota command

### PR DESCRIPTION
**Pull request by igoihman**
```
$ ./ocm list quota
RESERVED/ALLOWED    RESOURCE NAME                     RESOURCE TYPE               AVAILABILITY ZONE    CCS
0/15                addon-acm-operator                addon                       ANY                  false
0/15                addon-cluster-logging-operator    addon                       ANY                  false
0/15                addon-crw-operator                addon                       ANY                  false
0/15                addon-dba-operator                addon                       ANY                  false
0/15                addon-elasticsearch-operator      addon                       ANY                  false
0/15                addon-managed-api-service         addon                       ANY                  false
0/15                addon-managed-kafka               addon                       ANY                  false
0/15                addon-ocs-converged               addon                       ANY                  false
0/15                addon-open-data-hub               addon                       ANY                  false
0/15                addon-prow-operator               addon                       ANY                  false
0/15                addon-rhmi-operator               addon                       ANY                  false
0/7                 c5.2xlarge                        cluster.aws                 MULTI                false
0/7                 c5.2xlarge                        cluster.aws                 SINGLE               false
0/7                 c5.4xlarge                        cluster.aws                 MULTI                false
0/7                 c5.4xlarge                        cluster.aws                 SINGLE               false
0/7                 m5.2xlarge                        cluster.aws                 MULTI                false
0/7                 m5.2xlarge                        cluster.aws                 SINGLE               false
0/7                 m5.4xlarge                        cluster.aws                 MULTI                false
0/7                 m5.4xlarge                        cluster.aws                 SINGLE               false
0/10876             m5.xlarge                         cluster.aws                 ANY                  false
0/17                m5.xlarge                         cluster.aws                 MULTI                false
0/115               m5.xlarge                         cluster.aws                 SINGLE               false
0/10                m5.xlarge                         cluster.aws                 MULTI                true
1/10                m5.xlarge                         cluster.aws                 SINGLE               true
0/7                 r5.2xlarge                        cluster.aws                 MULTI                false
0/7                 r5.2xlarge                        cluster.aws                 SINGLE               false
0/7                 r5.4xlarge                        cluster.aws                 MULTI                false
0/7                 r5.4xlarge                        cluster.aws                 SINGLE               false
0/7                 r5.xlarge                         cluster.aws                 MULTI                false
0/7                 r5.xlarge                         cluster.aws                 SINGLE               false
0/2                 c5.2xlarge                        compute.node.aws            MULTI                false
0/2                 c5.4xlarge                        compute.node.aws            MULTI                false
0/2                 m5.2xlarge                        compute.node.aws            MULTI                false
0/2                 m5.4xlarge                        compute.node.aws            MULTI                false
0/27                m5.xlarge                         compute.node.aws            MULTI                false
0/7                 r5.2xlarge                        compute.node.aws            MULTI                false
0/2                 r5.4xlarge                        compute.node.aws            MULTI                false
0/2                 r5.xlarge                         compute.node.aws            MULTI                false
0/40                network                           network.loadbalancer.aws    MULTI                false
0/26500             gp2                               pv.storage.aws              MULTI                false
2/0                 m5.xlarge                         compute.node.aws            SINGLE               true
2/0                 custom-4-16384                    cluster.gcp                 SINGLE               true
4/0                 custom-4-16384                    compute.node.gcp            SINGLE               true
1/0                 custom-4-16384                    cluster.gcp                 SINGLE               false

```

### CODE REVIEW

**Comment by cben** [comment]
This doesn't follow the usual columnar outupt of most `ocm list` commands — is that intentional?

This prints `ResourceName` without `ResourceType`.  This makes cluster vs node quotas too confusing in the output:
```
$ ./ocm list quota | grep m5.xlarge
0/17 m5.xlarge MULTI
1/35 m5.xlarge SINGLE
0/10 m5.xlarge MULTI BYOC
0/10 m5.xlarge SINGLE BYOC
0/27 m5.xlarge MULTI
```
Including the type would help:
```
0/17 cluster.aws m5.xlarge MULTI
1/35 cluster.aws m5.xlarge SINGLE
0/10 cluster.aws m5.xlarge MULTI BYOC
0/10 cluster.aws m5.xlarge SINGLE BYOC
0/27 compute.node.aws m5.xlarge MULTI
```

If you're making a new command, I think it's worth extending this to include `--parameter=fetchRelatedResources=true` information?

cc @rawsyntax please advice how to make a user-friendly quota view command.

File: cmd/ocm/list/quota/cmd.go [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+			return fmt.Errorf("Failed to retrieve quota: %v", err)
+		}
+
+		// Display quota information:
+		fmt.Printf("Cluster quota for organization '%s' ID: '%s'\n",
+			orgResponse.Body().Name(), orgResponse.Body().ID())
+		quotasListResponse.Items().Each(func(quota *amv1.QuotaSummary) bool {
+			var byoc string
+			if quota.BYOC() {
+				byoc = " BYOC"
```
Comment (cben) [inline cmd/ocm/list/quota/cmd.go:107]: The account-manager field is still named "byoc", but in output I think we want to call this "CCS".

Comment (igoihman) [reply cmd/ocm/list/quota/cmd.go:107]: this is not a new command - I am just changing the command syntax. Specific enhancements will be done outside this change

Comment (cben) [reply cmd/ocm/list/quota/cmd.go:107]: I was thinking that as soon as we add `ocm list quota` with specific behavior, somebody might start using it and changing it later may come with question of backward compatibility.
While you have it on an unmerged branch, it's easier to re-imagine it however we want.
We can still do reviews step-by-step.  For the goal of copying `ocm account quota` exactly, this LGTM :100:.

This is not a strong argument, I'm OK with merging and iterating, your call.

File: cmd/ocm/list/quota/cmd.go [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+		orgID = userOrg.ID()
+	}
+
+	// Get connection
+	orgCollection := connection.AccountsMgmt().V1().Organizations().Organization(orgID)
+	orgResponse, err := orgCollection.Get().Send()
+	if err != nil {
+		return fmt.Errorf("Can't retrieve organization information: %v", err)
+	}
+	quotaClient := orgCollection.QuotaSummary()
```
Comment (rawsyntax) [inline cmd/ocm/list/quota/cmd.go:89]: quota summary is deprecated. This should use the quota cost endpoint

Comment (igoihman) [reply cmd/ocm/list/quota/cmd.go:89]: @rawsyntax can you point me to a quota cost usage example?

Comment (rawsyntax) [reply cmd/ocm/list/quota/cmd.go:89]: Yes I'll send you one

File: cmd/ocm/list/quota/cmd.go [hunk 3]
```diff
@@ -0,0 +1,129 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
```
Comment (tzvatot) [inline cmd/ocm/list/quota/cmd.go:2]: 2021

**Comment by tzvatot** [comment]
LGTM

**Pull request merged**